server/api.js

```diff
@@ -131,23 +134,11 @@ router.post("/energizers/:id/rate", async (req, res) => {
 
 		// Update the average rating of the energizer in the energizers table
 		await db.query(
-			"UPDATE energizer_ratings SET average_rate = $1 WHERE id = $2",
+			"UPDATE energizers SET average_rate = $1 WHERE id = $2",
```

Comment:
Does this column exist? Again I don't see this being used anywhere (it's calculate dynamically in the GET endpoint) so this UPDATE is pointless.

Reply:
yes this column exist , and its in use on Energiser detail page , to show average rate among with the rest of info on each energiser ,when we fetch an specific energiser with its ID , as we created an object for each energiser we needed a column to save the average in it , so when some on on home page clicks on most recent energiser to see more detail on it ,we could have the updated average rate
	const energizer = {
				"id": row.id,
				"name": row.name,
				"description": row.description,
				"submission_date": row["submission_date"],
				"url_address": row.url_address,
				"ratings": [],
				"average_rate": 0,
			};

Reply:
It's not a database column, the average_rate field in the JSON object is calculated by the API endpoint. The database column `average_rate` in `energizer_ratings` is never used, and it doesn't exist in the `energizers` table.